pkg/blockchain/registryAdmin.go

```diff
@@ -0,0 +1,83 @@
+package blockchain
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"fmt"
+	"time"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/xmtp/xmtpd/pkg/abis"
+	"github.com/xmtp/xmtpd/pkg/config"
+	"go.uber.org/zap"
+)
+
+/*
+*
+A NodeRegistryAdmin is a struct responsible for calling admin functions on the node registry
+*
+*/
+type NodeRegistryAdmin struct {
+	client   *ethclient.Client
+	signer   TransactionSigner
+	contract *abis.Nodes
+	logger   *zap.Logger
+}
```

Comment:
Is this used in production code, or only in unit tests?

Reply:
Great question. 

Unit tests right now, but I think we might want a CLI command to do admin actions at least in the testnet phase. So, not purely a test util.